static/js/admin/src/ResultObjectActionMenu.js

```diff
@@ -196,10 +196,15 @@ define('ResultObjectActionMenu', [ 'jquery', 'jquery-ui', 'StringUtilities',  'A
 		}
 
 		// Export actions
-		if (!isContentRoot) {
-			items["sepexport"] = "";
+		if ($.inArray('viewHidden', metadata.permissions) !== -1) {
+			items["export"] = {name: "Export", items: {}}
+
+			if (!isContentRoot && metadata.type !== 'File' && $.inArray('viewHidden', metadata.permissions) != -1) {
```

Comment:
You don't need to check for the 'viewHidden' permission again here or in the following conditional since this whole block is wrapped in a check already. It actually looks like you could merge together this conditional and the Export Member Order one, then wrap Export CSV with `!isContentRoot`